src/components/CIPipelineN/InputPluginSelect.tsx

```diff
@@ -171,6 +206,16 @@ export const InputPluginSelection = ({
                     tabIndex={selectedVariableIndex}
                     handleKeyDown={handleOnKeyDown}
                 />
+                {variableData?.variableType && variableData.variableType !== 'NEW' && (
+                    <Tippy
+                        content={'This is a variable. It will be replaced with the value during execution.'}
```

Comment:
we can directly write it as `content="This is a variable. It will be replaced with the value during execution."`

Reply:
resolved